support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper.java

```diff
@@ -160,7 +161,12 @@ public String getId() throws AbortException {
         if (build instanceof AbstractBuild) {
             return Collections.unmodifiableMap(((AbstractBuild<?,?>) build).getBuildVariables());
         } else { // not applicable
```

Comment:
Well, `AbstractBuild.getBuildVariables` is inapplicable, but I agree the comment could be confusing now.